resources/ftbquests/lang/zh_cn.lang

```diff
@@ -7,13 +7,13 @@ q.getting_started.welcome.desc=欢迎来到 Enigmatica 2: Expert - Extended！
 q.getting_started.wood.name=木材
 q.getting_started.wood.desc=你可能已经发现木板有些难做。\n在箱子中搜索物品，鼠标放在要搜的物品上然后按 §l§c§lY
 q.getting_started.important_nerfs.name=重要的削弱
-q.getting_started.important_nerfs.desc=§7相比 E2E 的削弱：§r\n\n- §n原版火把§r 与 §n石制火把§r 的光照等级被削弱，使用模组的。\n\n- FTB 区块声明 §4不会§r 防爆。
+q.getting_started.important_nerfs.desc=§7相比 E2E 的削弱：§r\n\n§2-§r §n原版火把§r与§n石制火把§r的光照等级被削弱，使用模组的。\n\n§2-§r FTB 区块声明§4不会§r防爆。
```

Comment:
Why you remved `§2-§r` here?

Reply:
Uh... I think I didn't delete it, I added it back.

Reply:
AI sometimes deletes and adds text somehow like this.
It takes me a lot of time to correct it.

Reply:
You need to use another AI i guess. I hear Claude or ChatGPT-4o doing good job with that.